beacon-chain/rpc/eth/light-client/helpers_test.go

```diff
@@ -1,616 +1,678 @@
-				},
-				AttestedHeader: &ethpbv2.LightClientHeaderContainer{
-					Header: &ethpbv2.LightClientHeaderContainer_HeaderAltair{
-						HeaderAltair: &ethpbv2.LightClientHeader{Beacon: &ethpbv1.BeaconBlockHeader{
-							Slot: 1000000,
-						}},
-					},
-				},
-				NextSyncCommitteeBranch: createNonEmptySyncCommitteeBranch(),
-				SignatureSlot:           9998,
-				FinalityBranch:          createNonEmptyFinalityBranch(),
-				FinalizedHeader: &ethpbv2.LightClientHeaderContainer{
-					Header: &ethpbv2.LightClientHeaderContainer_HeaderAltair{
-						HeaderAltair: &ethpbv2.LightClientHeader{Beacon: &ethpbv1.BeaconBlockHeader{
-							Slot: 9999,
-						}},
-					},
-				},
-			},
-			expectedResult: true,
-		},
-		{
-			name: "none of the above conditions are met and new signature's slot is greater than old signature's slot",
-			oldUpdate: &ethpbv2.LightClientUpdate{
-				SyncAggregate: &ethpbv1.SyncAggregate{
-					SyncCommitteeBits: []byte{0b00111100, 0b1}, // [0,0,1,1,1,1,0,0]
-				},
-				AttestedHeader: &ethpbv2.LightClientHeaderContainer{
-					Header: &ethpbv2.LightClientHeaderContainer_HeaderAltair{
-						HeaderAltair: &ethpbv2.LightClientHeader{Beacon: &ethpbv1.BeaconBlockHeader{
-							Slot: 1000000,
-						}},
-					},
-				},
-				NextSyncCommitteeBranch: createNonEmptySyncCommitteeBranch(),
-				SignatureSlot:           9998,
-				FinalityBranch:          createNonEmptyFinalityBranch(),
-				FinalizedHeader: &ethpbv2.LightClientHeaderContainer{
-					Header: &ethpbv2.LightClientHeaderContainer_HeaderAltair{
-						HeaderAltair: &ethpbv2.LightClientHeader{Beacon: &ethpbv1.BeaconBlockHeader{
-							Slot: 9999,
-						}},
-					},
-				},
-			},
-			newUpdate: &ethpbv2.LightClientUpdate{
-				SyncAggregate: &ethpbv1.SyncAggregate{
-					SyncCommitteeBits: []byte{0b00111100, 0b1}, // [0,0,1,1,1,1,0,0]
-				},
-				AttestedHeader: &ethpbv2.LightClientHeaderContainer{
-					Header: &ethpbv2.LightClientHeaderContainer_HeaderAltair{
-						HeaderAltair: &ethpbv2.LightClientHeader{Beacon: &ethpbv1.BeaconBlockHeader{
-							Slot: 1000000,
-						}},
-					},
-				},
-				NextSyncCommitteeBranch: createNonEmptySyncCommitteeBranch(),
-				SignatureSlot:           9999,
-				FinalityBranch:          createNonEmptyFinalityBranch(),
-				FinalizedHeader: &ethpbv2.LightClientHeaderContainer{
-					Header: &ethpbv2.LightClientHeaderContainer_HeaderAltair{
-						HeaderAltair: &ethpbv2.LightClientHeader{Beacon: &ethpbv1.BeaconBlockHeader{
-							Slot: 9999,
-						}},
-					},
-				},
-			},
-			expectedResult: false,
-		},
-	}
 
-	for _, testCase := range testCases {
-		t.Run(testCase.name, func(t *testing.T) {
-			result, err := IsBetterUpdate(testCase.newUpdate, testCase.oldUpdate)
-			assert.NoError(t, err)
-			assert.Equal(t, testCase.expectedResult, result)
+	params.SetupTestConfigCleanup(t)
+	config := params.BeaconConfig()
+
+	t.Run("new has supermajority but old doesn't", func(t *testing.T) {
```

Comment:
The pattern that we had is also fine. Can you go back to it so that the diff is smaller?

Reply:
It's not possible to create these updates and set their values with wrapper setters, using the previous pattern.